recipes/r-downscaler/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge,r,defaults
```

Comment:
Don't do this... packages in r/defaults are build against older R versions, so don't help here.